src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.cs

```diff
@@ -1544,6 +1544,59 @@ public void CanBindOnParametersAndProperties_RecordWithArrayConstructorParameter
             Assert.Equal(new string[] { "a", "b", "c" }, options.Array);
         }
 
+        /// <summary>
+        /// Test binding a value parsable types to null configuration values.
+        /// The test ensure the binding will succeed without exceptions and the value will be null (not set).
+        /// </summary>
+        [Fact]
+        public void BindToKnownParsableTypesWithNullValueTest()
+        {
+            string jsonConfig = @"{
+                            ""ParsableValuesClass"": {
```

Comment:
Should we also test the empty string case?

Reply:
good idea. added!